source/backend/api/Areas/Property/Models/Search/PropertyFilterModel.cs

```diff
@@ -49,10 +55,32 @@ public PropertyFilterModel(Dictionary<string, Microsoft.Extensions.Primitives.St
             var filter = new Dictionary<string, Microsoft.Extensions.Primitives.StringValues>(query, StringComparer.OrdinalIgnoreCase);
 
             this.Sort = filter.GetStringArrayValue(nameof(this.Sort));
+            var tempSort = this.Sort.ToList();
+
+            // Convert sort to db format
+            for (int i = 0; i < this.Sort.Length; i++)
+            {
+                if (tempSort[i].StartsWith("Location"))
+                {
+                    tempSort[i] = tempSort[i].Replace("Location", "Address.MunicipalityName");
+                }
+                if (tempSort[i].StartsWith("Ownership"))
+                {
+                    // The order will affect the display in the frontend. For now in alphabetical order.
+                    // i.e. [Core Inventory, Disposed, Other Interest, Property of Interest]
+                    var direction = this.Sort[i].Split(' ')[1];
```

Comment:
is this safe (ie. we always have a direction of asc or desc)?

Reply:
yes, the sort expects a direction to be able to be process later